test/integration/log_system.cc

```diff
@@ -1523,7 +1523,7 @@ TEST_F(LogSystemTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(LogResources))
   // Recorded models should exist
   EXPECT_GT(entryCount(recordPath), 2);
   EXPECT_TRUE(common::exists(common::joinPaths(recordPath, homeFake,
-      ".ignition", "fuel", "fuel.ignitionrobotics.org", "openrobotics",
+      ".gz", "fuel", "fuel.ignitionrobotics.org", "openrobotics",
```

Comment:
As mentioned on the `fuel-tools` PR, let's revert these for now

Reply:
https://github.com/gazebosim/gz-sim/pull/1518/commits/3417cef701d64d734dadaf3d2bafd1754e89b383